### DISCUSSION TITLE
en/blog/istio-service-mesh-book/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#305] [Announcements]
# en/blog/istio-service-mesh-book/

By the Cloud Native Community(China)

https://jimmysong.io/en/blog/istio-service-mesh-book/

**Comment by ssignori76** [comment]
The book will be translated in English?

**Reply by rootsongjc** [reply]
I thought it might be possible to translate some of the chapters in this book into English, but not the whole book.
